.github/workflows/deploy.yml

```diff
@@ -17,9 +16,7 @@ jobs:
       postgres_unit_testing: true
       db_name: fsd_app_store_test
       deploy_to_dev: false
```

Comment:
In account store we now set deploy_to_dev to True. Are we okay setting this to True for Application store @gio-karunakaran?

Reply:
I think we're still undecided about this, tech team will have a separate discussion to agree on whether or not we should change this

Reply:
https://digital.dclg.gov.uk/jira/browse/FS-898